packages/simcore-sdk/src/simcore_sdk/node_ports_common/dbmanager.py

```diff
@@ -110,45 +113,59 @@ async def write_ports_configuration(
         log.debug(message)
 
         node_configuration = json.loads(json_configuration)
-        async with DBContextManager(self._db_engine) as engine:
-            async with engine.acquire() as connection:
-                # update the necessary parts
-                await connection.execute(
-                    # FIXME: E1120:No value for argument 'dml' in method call
-                    # pylint: disable=E1120
-                    comp_tasks.update()
-                    .where(
-                        and_(
-                            comp_tasks.c.node_id == node_uuid,
-                            comp_tasks.c.project_id == project_id,
-                        )
-                    )
-                    .values(
-                        schema=node_configuration["schema"],
-                        inputs=node_configuration["inputs"],
-                        outputs=node_configuration["outputs"],
-                        run_hash=node_configuration.get("run_hash"),
+        async with DBContextManager(
+            self._db_engine
+        ) as engine, engine.acquire() as connection:
+            # update the necessary parts
+            await connection.execute(
+                # FIXME: E1120:No value for argument 'dml' in method call
+                # pylint: disable=E1120

```

Comment:
bazinga!  :-) you have just inherited a FIXME. :-)

Please help sanitizing old code. Could you please at least reveal waht `pylint: disable=E1120` means? If possible, please review the FIXME and see
a) if it is up-to-date
b) if there is an issue associated to it already. If not please update note/create issue.
thx

Reply:
That would be no value for argument. But apparently this is no longer the case. Removed it.